app/src/main/java/com/framgia/soundclound/util/Constant.java

```diff
@@ -39,5 +40,10 @@
     public static final String EXTRA_TRACK = "com.frangia.soundcloud.EXTRA_TRACK";
     public static final int INDEX_DEFAULTE = 1;
     public static final String FILE_EXTENTION = ".mp3";
-}
 
+    public static int FOREGROUND_SERVICE = 101;
+    public static String ACTION_MAIN = "com.framgia.soundclound.action.main";
```

Comment:
ERROR: [MALICIOUS_CODE][MS_SHOULD_BE_FINAL] com.framgia.soundclound.util.Constant.ACTION_MAIN isn't final but should be
http://findbugs.sourceforge.net/bugDescriptions.html#MS_SHOULD_BE_FINAL